### PR TITLE
ppvernac fixes (univdecl incorrect printing and attribute spacing)

### PR DESCRIPTION
**Pull request by SkySkimmer**
Depends 
- https://github.com/coq/coq/pull/19073

### CODE REVIEW

File: vernac/ppvernac.ml [hunk 1]
```diff
@@ -83,24 +88,30 @@ let pr_cumul_univdecl_instance l extensible =
 
 let pr_univdecl_constraints l extensible =
   if List.is_empty l && extensible then mt ()
-  else str"|" ++ spc () ++ prlist_with_sep (fun () -> strbrk" | ") pr_uconstraint l ++
+  else strbrk" | " ++ prlist_with_sep (fun () -> strbrk", ") pr_uconstraint l ++
```
Comment (silene) [inline vernac/ppvernac.ml:91]:
```suggestion
  else strbrk" | " ++ prlist_with_sep pr_comma pr_uconstraint l ++
```

Also, unless you insist on breaking before the pipe, you can use `pr_spcbar ()` in place of `strbrk" | "`.

Comment (SkySkimmer) [reply vernac/ppvernac.ml]: changed

**Comment by proux01** [comment]
@coqbot run full ci

**Review by proux01 — APPROVED** [review]

**Comment by proux01** [comment]
@coqbot merge now

**Pull request merged**